deps/openssl/openssl_common.gypi

```diff
@@ -32,6 +32,7 @@
       'cflags': [
         '-W3', '-wd4090', '-Gs0', '-GF', '-Gy', '-nologo','/O2',
       ],
+      'msvs_disabled_warnings': [4090],
```

Comment:
As far as I can tell (setting the `msbuild_args=/clp:Verbosity=n` environment variable before calling `vcbuild` to get verbose command lines being passed to `cl.exe`) the above `cflags` (containing `-wd4090` which should suppress the warning) is being ignored. I've left the `cflags` alone for now as I can't be sure it isn't used in non-msvs scenarios on Windows (Ninja?).

Reply:
For reference, OpenSSL's own build configuration ignores the same warning:
https://github.com/nodejs/node/blob/be055d103c85d4b716f3013f8c63b39c7dc0425b/deps/openssl/openssl/Configurations/10-main.conf#L1199-L1202